packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -882,10 +882,14 @@ class AnimatedAlign extends ImplicitlyAnimatedWidget {
     Key key,
     @required this.alignment,
     this.child,
+    this.heightFactor = 1.0,
+    this.widthFactor = 1.0,
     Curve curve = Curves.linear,
     @required Duration duration,
     VoidCallback onEnd,
   }) : assert(alignment != null),
+        assert(widthFactor == null || widthFactor >= 0.0),
+        assert(heightFactor == null || heightFactor >= 0.0),
```

Comment:
@hansmuller just astutely noticed the Align-ment (😆 ) here is off. Can you fix this nit before we merge?

```suggestion
       assert(widthFactor == null || widthFactor >= 0.0),
       assert(heightFactor == null || heightFactor >= 0.0),
```

Reply:
Hahaha 😆  done